packages/modals/src/styled/StyledTooltipModalFooter.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'modals.tooltip_modal.footer';
+
+export const StyledTooltipModalFooter = styled.div.attrs({
```

Comment:
Looks like https://github.com/zendeskgarden/react-components/blob/main/packages/modals/src/styled/StyledFooter.ts could be factored to extend this styled component.

Reply:
Same comment as https://github.com/zendeskgarden/react-components/pull/828#discussion_r469426145